data/sql/updates/pending_db_world/fix-issue-15534.sql

```diff
@@ -0,0 +1,14 @@
+
+SET @LOOTTHISTLEBOAR := 1985; -- Skinning loot template for Thistle Boar
+SET @LOOTSCORPIDWORKER := 3124; -- Skinning loot template for Scorpid Worker
+SET @LIGHTLEATHER := 2318; -- Light Leather
+SET @RUINEDSCRAPS := 2934; -- Ruined Leather Scraps
+
+-- Setting skinloot as 0 for creature that should not be skinnable
+UPDATE `creature_template` SET `skinloot` = 0 WHERE `entry` IN (1984, 1985, 2032, 3124);
+
+-- Removing now unused skinning loot template to keep the DB cleaned up
```

Comment:
Do the remaining entries who skin template loot wasn't removed now still have values in this table even if skinloot in creature template was set to 0?

Reply:
Sorry, I don't think I understood. Before this PR creature with entry 1984 and 2032 had skinloot = 100002. I didn't delete it because it's linked to many other creature. Instead creature 1985 had skinloot 1985 and creature 3124 had skinloot 3124: both of them were used only by that creature so I proceeded to delete'em.

Reply:
Nvm I'll check when I get home as I'm not that familiar with how the two tables are linked. FK and PK wise